elements/document/nuxeo-document-create.js

```diff
@@ -50,7 +50,10 @@ Polymer({
         --paper-dialog-scrollable: {
           padding: 0;
           overflow-x: hidden;
-        }
+        };
+        --nx-button-icon: {
+          margin-inline-end: 0;
+        };
```

Comment:
Couldn't we just replace this with
```suggestion
        --nuxeo-button-icon-margin-end: 0;
```
?

Reply:
@semisse do you know if we really need to have `margin-inline-end: var(--nuxeo-button-icon-margin-end, 4px);` on every button by default on base.js?

Reply:
@Gabez0r I tried that, but it doesn't work. About the use of `margin-inline-end` for every button on default, we were discussing that with @andreacornaglia as well to understand the use cases.

Reply:
The purpose of having the `margin-inline-end: var(--nuxeo-button-icon-margin-end, 4px);` for every button by default is to support horizontal buttons with an icon and text. It seems to me that the base class should not define any margin at all, and we should add another class to be used for those buttons. wdyt?

Reply:
I agree. I a button with an icon does not have a label, or have the label on the bottom, this doesn't make sense anymore.

Reply:
Same for me. [This should be removed](https://github.com/nuxeo/nuxeo-web-ui/blob/master/themes/base.js#L560) and handled for specific use cases if necessary.

Reply:
As far as I understoond, only four elements seem to need this:

- nuxeo-user-group-management
- nuxeo-group-management
- nuxeo-user-management
- nuxeo-user-profile

Reply:
I'd be ok width just removing the `margin-inline-end: var(--nuxeo-button-icon-margin-end, 4px);` from base.js, since this doesn't seem to be a common case...

Reply:
Should we add a new class for the users and group management elements?

Reply:
Yes, I agree with adding a new class for both the users and group management elements. I'll create a separate ticket for handling this and assign it to me, so I can tackle both at the same time.